### ISSUE TITLE
App crashes when receiving incoming call on Android

### ISSUE THREAD

**Issue by Yahllil** [cybex-dev/twilio_voice#109]
Hello,
We are using this package for more than a year, and recently we experience a strange bug on Android when receiving an incoming call.

When debugging, we can see this error message printed to the console before the app crashes:

`Caused by: java.lang.IllegalArgumentException: com.package.id: Targeting S+ (version 31 and above) requires that one of FLAG_IMMUTABLE or FLAG_MUTABLE be specified when creating a PendingIntent.`

and

`Strongly consider using FLAG_IMMUTABLE, only use FLAG_MUTABLE if some functionality depends on the PendingIntent being mutable, e.g. if it needs to be used with inline replies or bubbles.`

```
E/TSLocationManager(  416): 	at android.app.PendingIntent.checkFlags(PendingIntent.java:382)
E/TSLocationManager(  416): 	at android.app.PendingIntent.getActivityAsUser(PendingIntent.java:465)
E/TSLocationManager(  416): 	at android.app.PendingIntent.getActivity(PendingIntent.java:451)
E/TSLocationManager(  416): 	at android.app.PendingIntent.getActivity(PendingIntent.java:415)
E/TSLocationManager(  416): 	at com.twilio.twilio_voice.IncomingCallNotificationService.createNotification(IncomingCallNotificationService.java:77)
E/TSLocationManager(  416): 	at com.twilio.twilio_voice.IncomingCallNotificationService.setCallInProgressNotification(IncomingCallNotificationService.java:305)
E/TSLocationManager(  416): 	at com.twilio.twilio_voice.IncomingCallNotificationService.handleIncomingCall(IncomingCallNotificationService.java:292)
E/TSLocationManager(  416): 	at com.twilio.twilio_voice.IncomingCallNotificationService.onStartCommand(IncomingCallNotificationService.java:42)
E/TSLocationManager(  416): 	at android.app.ActivityThread.handleServiceArgs(ActivityThread.java:5092)
E/TSLocationManager(  416): 	... 9 common frames omitted
I/Process (  416): Sending signal. PID: 416 SIG: 9
```

And then the app crashes.
How can we fix that?

**Comment by cybex-dev** [comment]
Hi @Yahllil I'll push out an update to resolve this next week.

I've resolved this using a fork, will merge changes.

**Comment by cybex-dev** [comment]
Merged in #116
